doc/source/getting_started/overview.rst

```diff
@@ -174,4 +174,3 @@ License
 -------
 
 .. literalinclude:: ../../../LICENSE
-
```

Comment:
I think the blank line at the end of the file was fine, could you revert this?

Reply:
I have added the space back in the latest commit.